src/app/sign_up/sign_up.html

```diff
@@ -0,0 +1,36 @@
+<section class="sign_up">
+  <h1 class="sign-up-header">sign up</h1>
+  <div class="form-container">
+    <form name="signUpForm" ng-submit="signUpForm.$valid && signUpCtrl.createUser()" novalidate>
+      <label for="name" class="form-label" ng-hide="(signUpForm.$submitted || signUpForm.name.$touched) && signUpForm.name.$error.required">name</label>
```

Comment:
I'm not seeing the app running, but are you trying to hide the label depending if the user has touched or filled the input?